### ISSUE TITLE
Add support for mypy and typing

### ISSUE THREAD

**Issue by dheerajreal** [googlemaps/google-maps-services-python#444]
currently mypy complains on using this library.
This can be fixed by asking mypy to ignore typing for this library (using `# type: ignore`)

Or type hints can be added in the library

**Comment by soulshake** [comment]
> This issue has been automatically marked as stale because it has not had recent activity. Please comment here if it is still valid so that we can reprioritize. Thank you!

Not stale!

**Comment by scottynoshotty** [comment]
Should be fixed

**Comment by shaer2** [comment]
Yes, please I need help with that‫في 18‏/06‏/2023 الساعة 3:53 م، كتب/كتبت ‏stale[bot] ***@***.***>:‬﻿
This issue has been automatically marked as stale because it has not had recent activity. Please comment here if it is still valid so that we can reprioritize. Thank you!

**Comment by andrecuellar** [comment]
+1
